go/vt/vttablet/tabletserver/schema/tracker.go

```diff
@@ -231,10 +231,15 @@ func (tr *Tracker) saveCurrentSchemaToDb(ctx context.Context, gtid, ddl string,
 	}
 	defer conn.Recycle()
 
+	// We serialize a blob here, encodeString is for strings only
+	// and should not be used for binary data.
```

Comment:
Found this with cleaning up the helpers, that here we were encoding a blob as a string which shouldn't be done.